src/Microsoft.Identity.Web.Certificate/SignedAssertionFilePathCredentialsLoader.cs

```diff
@@ -36,13 +36,14 @@ public async Task LoadIfNeededAsync(CredentialDescription credentialDescription,
                 {
                     // Given that managed identity can be not available locally, we need to try to get a
                     // signed assertion, and if it fails, move to the next credentials
-                    _= await signedAssertion!.GetSignedAssertionAsync(null);
+                    _ = await signedAssertion!.GetSignedAssertionAsync(null);
                     credentialDescription.CachedValue = signedAssertion;
                 }
                 catch (Exception)
                 {
                     credentialDescription.Skip = true;
-                }
+                    throw;
```

Comment:
If we throw at all stages, how will the chained credentials work?

Reply:
The orchestrator catches exceptions. You can see all the other loaders having this problem.

Reply:
Silently failing here means you'll get a strange from MSAL, because you'll assume the credential worked and you'll use the `credentialDescription.CachedValue` which is null.

Reply:
But I can undo this if you like.

Reply:
that's not what I'm saying.
I just want to understand the new behavior and test it

Reply:
I'll run end to end tests.

Reply:
I can run a manual test @jmprieur . How about this: 

- cred chain of { Kubernetes FIC, certificate } 
- the Kubernetes FIC will fail 

Expected: the certificate will be attempted.

Reply:
Done^^